environment.yml

```
@@ -0,0 +1,15 @@
+name: t21-thu-fatiando
```

The Slack channel has been renamed to `t21-thurs-harmonica`, so we can use the same name for the environment.
```suggestion
name: t21-thurs-harmonica
```